code/game/objects/items/chainsaw.dm

```diff
@@ -31,7 +31,7 @@
 /obj/item/chainsaw/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/butchering, 30, 100, 0, 'sound/weapons/chainsawhit.ogg', TRUE)
-	AddComponent(/datum/component/two_handed, require_twohands=TRUE, block_power_unwielded=block_power, block_power_wielded=block_power)
+	AddComponent(/datum/component/two_handed, require_twohands=TRUE, block_power_unwielded=block_power, block_power_wielded=block_power,force_unwielded = force)
```

Comment:
I believe the thing that needs to be set is force_wielded rather than force_unwielded, although maybe both.

Reply:
actually this gets really fucky if its dropped while on, since it will get its force set back to the low damage force and still remain on. This needs some extra login on dropped() to turn it off or some serious fuckery with the two_handed component.

Reply:
That'd just be redundant or could cause an issue on it's own, the problem was that due to` force_unwielded` being nothing the chainsaw would just get set to `force = 0` when droppped (unweilded) I have tried it out in game and works fine now

Reply:
If dropped while on the force of the chainsaw at that point is of it on and is not changed

Reply:
I just tested it and you are exactly right, fucking hell

Reply:
The check to see if the value was null or not was not properly done which was causing the issue, fixed that which was much simpler